### ISSUE TITLE
Add card from mail [$15]

### ISSUE THREAD

**Issue by laurentiu2** [nextcloud/deck#392]
If a new mail arrives at a pre-determined mail address, transform it in a new card with the subject, message, sender, IP, etc.

This way Deck can be used as a simple customer support system.

I'm using Asana right now for this (with some services enabled to read mail, filter them after certain rules, etc) but I'd certainly prefer using NC and Deck for security reasons.

Regards,
Laurentiu

**Comment by juliushaertl** [comment]
That require way to much logic inside the deck app. Once #83 is ready, automatic card creation of incoming emails can be achieved with some additional backend script. But I don't see this landing directly inside deck.

**Comment by laurentiu2** [comment]
@juliushaertl I'd be glad to back this feature via Bountysource if someone is willing to address it. Maybe you'll leave it open as feature request.

**Comment by srbaker** [comment]
I am hoping to finish up #83 in the next few weeks, and will need "do things by email" for a few other projects. I'd be interested in working on this.

**Comment by juliushaertl** [comment]
@laurentiu2 Ok, fine. Feel free to add your bounty here. :+1:

**Comment by hmerk** [comment]
@srbaker 
Any Progress on your implementation ?

**Comment by putt1ck** [comment]
@hmerk if not we'll be sorting a solution built around the API

**Comment by juliushaertl** [comment]
The idea is to have this done by the API, the issue is just open to collect some bounty for it. @putt1ck If you are building something, having that available as an open solution would of course be awesome.

**Comment by srbaker** [comment]
@hmerk I had an old hack working, but it doesn't work against latest. I can pull again and do it against API but if @putt1ck has a current solution, they might be quicker.

**Comment by putt1ck** [comment]
We'll definitely get to doing an API-based solution - but if anyone else has the interest they might get done before us...

**Comment by putt1ck** [comment]
@juliushaertl We've got working code, but have some questions...

**Comment by juliushaertl** [comment]
Let me close this as there is work being done in https://github.com/putt1ck/mail2deck and this integration is out of scope of the deck repo. @putt1ck I guess you can apply for the bounty on this one ;)

**Comment by aktasfatih** [comment]
Hi,

just wanted to say this feature is developed in my own app Kanso. Feel free to check it out please 😄 

https://apps.nextcloud.com/apps/kanso

Docs are in https://github.com/aktasfatih/kanso/blob/main/docs/email-intake.md

<img width="519" height="804" alt="Image" src="https://github.com/user-attachments/assets/b7b7589d-5bb8-4dc0-aef5-c0bf5025469e" />
